.github/workflows/verify.yml

```diff
@@ -0,0 +1,33 @@
+name: verify
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          version: latest
+          only-new-issues: true
+
+  lint_markdown:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: read
+    steps:
+    - uses: actions/checkout@v3
+    - uses: dorny/paths-filter@v2
+      id: changes
+      with:
+        filters: |
+          md:
+            - 'README.md'
```

Comment:
It might make sense to run this when `.github/workflows/verify.yml` has been modified as well.

Reply:
Why? I don't get it.

Reply:
If we decide to change the workflow, or bump up the version of markdownlint-cli2, we would want to run the check, even if the markdown files in the repo haven't changed.